certbot/storage.py

```diff
@@ -833,6 +833,18 @@ def new_lineage(cls, lineagename, cert, privkey, chain, cli_config):
             logger.debug("Writing full chain to %s.", target["fullchain"])
             f.write(cert + chain)
 
+        # Write a README file to the live directory
+        readme_path = os.path.join(live_dir, "README")
+        with open(readme_path, "w") as f:
+            logger.debug("Writing README to %s.", readme_path)
+            f.write("This directory contains your keys and certificates.\n\n"
+                    "`privkey.pem` is the private key for your certificate.\n"
+                    "`fullchain.pem` is the certificate file used in modern server software.\n"
+                    "`chain.pem` is used for OCSP stapling in Nginx >=1.3.7.\n\n"
+                    "We recommend not moving these files. "
+                    "For more information, see the Certbot User Guide "
+                    "at https://certbot.eff.org/docs/using.html.\n")
```

Comment:
Concluded that we do need *some* docs for `cert.pem`, not least because some modern server software probably still needs it (though not Nginx and Apache).